system/CodeIgniter.php

```diff
@@ -310,8 +310,6 @@ private function configureKint(): void
      * makes all of the pieces work together.
      *
      * @return ResponseInterface|void
-     *
-     * @throws RedirectException
```

Comment:
since the method has `LogicException` throw, how about change to `@throws LogicException` ?

Reply:
I think we should write Exceptions in `@throws` which should be caught in the client code.
LogicException should not be caught.

Reply:
I think adding `@throws` doesn't mean it have to be caught in client code. It's a way of explicitly telling that calling the function may throw this exception.

Reply:
@paulbalandan Do you think we should write all Exceptions that may be thrown?

Reply:
Not really *all*. Only those that are meant to be checked by the framework and/or user. Please read this [PHPStan article](https://phpstan.org/blog/bring-your-exceptions-under-control) for details.

Reply:
I read the article.
I think checked exceptions should be caught. So we should write only checked exceptions in `@thrown`.
Something wrong with my understanding?